src/core/homepage_elements/journals_and_html/forms.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+
+from press import models
+from journal import models as jm
+from django_summernote.widgets import SummernoteWidget
+
+
+class FeaturedJournalsForm(forms.ModelForm):
```

Comment:
There is a `FeaturedJournalsForm` in `src/core/homepage_elements/journals/forms.py` already, which also controls `random_featured_journals` and controls `models.Press`